src/Examine.Lucene/ExamineReplicator.cs

```diff
@@ -120,7 +123,10 @@ public void StartIndexReplicationOnSchedule(int milliseconds)
         private void SourceIndex_IndexCommitted(object sender, EventArgs e)
         {
             var index = (LuceneIndex)sender;
-            _logger.LogDebug("{IndexName} committed", index.Name);
+            if (_logger.IsEnabled(LogLevel.Debug))
```

Comment:
In most logging cases like this there's no need to do the check since this is not allocating anything since these are string constants. In the example above the logging check is necessary since there's some string concatenation happening but I think for the remaining cases there's no need to check.

Reply:
Hi @Shazwazza. For consistency I applied this pattern to all LogDebug calls. This follows the guidance here: https://docs.microsoft.com/en-us/dotnet/core/extensions/high-performance-logging#log-level-guarded-optimizations

Reply:
Okidoke, that is a fair call 👍